client/src/pages/LandingPage.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "../../index.css";
```

Comment:
![image](https://user-images.githubusercontent.com/48932159/218283839-bc1f4007-199a-4381-bf8e-818babd5e2ba.png)

Hey @TrizzIzBizy, it looks like VSCode did not auto fix the `index.css` import inside of `client/src/pages/LandingPage.js`. Once this error is fixed, issue #202 will be completed. Great work!

Reply:
@TrizzIzBizy hello! Just checking in on this. Please let us know if you'd like some help!